core/src/edu/asu/c3simulator/screens/EmploymentScreen.java

```diff
@@ -70,8 +73,9 @@ public void clicked(InputEvent event, float x, float y)
 			employeePosition.setText("Position: " + employee.getPosition());
 			employeePayField.setText("" + employee.getActualHourlyWage());
 			employeePreferredHourlyWage.setText("Income Preference: $"
-					+ employee.getPreferredHourlyWage() + " / hr");
-			employeeMorale.setText("Morale: " + employee.getMorale() * 100 + " %");
+					+ Math.round(employee.getPreferredHourlyWage()) + " / hr");
```

Comment:
What happens if an employee makes 15.5 per hour? Do we want to support the display of fractional amounts, or require integer wages?